### PR TITLE
Fixed #1341 -- Updated info about past and present Django Fellows.

### LINKED ISSUE #1341: Update list of fellows on the fundraising page.
List of fellows on the [fundraising page](https://www.djangoproject.com/fundraising/) is outdated. Natalia replaced Carlton as a Fellow in April, 2023 (see https://www.djangoproject.com/weblog/2023/mar/31/welcome-our-new-fellow-natalia-bidart/).

### PR DESCRIPTION
**Pull request by nessita**

### CODE REVIEW

File: djangoproject/templates/fundraising/index.html [hunk 1]
```diff
@@ -198,6 +199,12 @@ <h2 id="who-are-the-django-fellows">Who are the Django Fellows?</h2>
 
       <p>Former Django Fellows:</p>
 
+      <p><strong><a href="https://github.com/carltongibson">Carlton Gibson</a>
+        (2018-2023)</strong> - a longtime Django user, core contributor to
+        Django REST Framework, maintainer of Django Filter and Django Crispy
+        Forms, and a contributor to many other packages in the Django ecosystem.
+        Carlton <a href="https://www.djangoproject.com/weblog/2018/jan/12/dsf-welcomes-carlton-gibson-its-newest-fellow/">
+        began as a part-time Fellow in January 2018</a>.</p>
```
Comment (felixxm) [inline djangoproject/templates/fundraising/index.html:207]: I'd add a sentence about retirement as we did for Tim, e.g. 
```suggestion
        Carlton began as a part-time Fellow in January 2018.
        In 2023 Carlton retired after five years of service.</p>
```

**Review by CuriousLearner — APPROVED** [review]
LGTM 🌮

File: djangoproject/templates/fundraising/index.html [hunk 2]
```diff
@@ -198,6 +199,12 @@ <h2 id="who-are-the-django-fellows">Who are the Django Fellows?</h2>
 
       <p>Former Django Fellows:</p>
 
+      <p><strong><a href="https://github.com/carltongibson">Carlton Gibson</a>
+        (2018-2023)</strong> - a longtime Django user, core contributor to
+        Django REST Framework, maintainer of Django Filter and Django Crispy
+        Forms, and a contributor to many other packages in the Django ecosystem.
+        Carlton began as a part-time Fellow in January 2018.
```
Comment (CuriousLearner) [inline djangoproject/templates/fundraising/index.html:206]: Any specific reason to remove [this](https://www.djangoproject.com/weblog/2018/jan/12/dsf-welcomes-carlton-gibson-its-newest-fellow/) link here?

Comment (nessita) [reply djangoproject/templates/fundraising/index.html:206]: I was following the previous review comment (and suggestion) from @felixxm.

Comment (felixxm) [reply djangoproject/templates/fundraising/index.html:206]: IMO it's no longer needed.

**Comment by felixxm** [comment]
@nessita Thanks :+1:

**Comment by felixxm** [comment]
Deployed https://www.djangoproject.com/fundraising/#who-are-the-django-fellows.

**Pull request merged**